pkgs/tools/networking/smartdns/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchFromGitHub, openssl, fetchpatch }:
+
+stdenv.mkDerivation rec {
+  pname = "smartdns";
+  version = "30"; # This would be used later in the next release as the FHS commit would be integrated into realse 31.
```

Comment:
So you need only [this change](https://github.com/pymumu/smartdns/commit/3ad7cd7f454eec2fbdf338c0eb0541da301f1e73) after tag 30?

If so, I'd make this clearer by using `rev = "Release${version}";` and adding a `fetchpatch` that will pull https://github.com/pymumu/smartdns/commit/3ad7cd7f454eec2fbdf338c0eb0541da301f1e73.patch .

Reply:
No, but this would be disgusting to change the way of compiling and installing. By now, for release 31, we only need to change the `rev` and the` version`

Reply:
It would be very soon to get the release 31 released from upstream, so I think it would be easier for me :)

Reply:
OK, no big deal I guess..